genref/config.yaml

```diff
@@ -56,90 +61,108 @@ apis:
       - k8s.io/controller-manager/config/v1alpha1
 
   - name: kube-proxy-config
+    title: kube-proxy configuration (v1alpha1)
     package: k8s.io/kube-proxy
     path: config/v1alpha1
 
   - name: metrics
+    title: ""
     package: k8s.io/metrics
     path: pkg/apis/metrics/v1alpha1
 
   - name: metrics
+    title: ""
     package: k8s.io/metrics
     path: pkg/apis/metrics/v1beta1
 
   - name: client-authentication
+    title: Client authentication (v1alpha1)
```

Comment:
nit: Does it make sense to capitalize the titles (the site typically does capitalize titles, but this case seems different) or leave lowercase?

Reply:
OK.